projects/htslib/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://www.htslib.org/"
+primary_contact: "rmd@sanger.ac.uk"
+auto_ccs:
+        - "aw7@sanger.ac.uk"
```

Comment:
Are these google accounts? You probably can't access the testcases without one.

Reply:
They should all be linked to a Google account.